src/sentry/static/sentry/app/components/dropdownControl.tsx

```diff
@@ -20,7 +20,21 @@ type DefaultProps = {
   menuWidth: string;
 };
 
+type ChildrenArgs = {
+  isOpen: boolean;
+  getMenuProps: GetMenuPropsFn;
+};
+
+type ButtonArgs = {
+  isOpen: boolean;
+  getActorProps: GetActorPropsFn;
+};
+
 type Props = DefaultProps & {
+  children:
+    | ((args: ChildrenArgs) => React.ReactElement)
+    | React.ReactElement
+    | Array<React.ReactNode>;
```

Comment:
Could this be `React.ReactNode | React.ReactNode[]` or do you only want a single element and not text?

Reply:
As `React.ReactNode` contains functions, I am getting a type conflict. That is why I can't use it

Reply:
I updated the types:

```
children:
    | ((args: ChildrenArgs) => React.ReactElement)
    | React.ReactElement
    | Array<React.ReactElement>;
```